### PR TITLE
#164829201 send user verification link via email

### PR DESCRIPTION
**Pull request by Sm0k3s**
#### Description
When a user signs up an email is instantly sent to the email they provided. It contains the verification link that is unique to them and can only be used once. 

#### Type of change
- [ ] Bug fix (nonbreaking change which fixes an issue)
- [x] New feature (nonbreaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

#### How Has This Been Tested?
Please describe the tests that you ran to verify your changes
- [x] End to end
- [x] Integration

#### Checklist:
- [ ] My code follows the style guide for this project
- [ ] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

#### Pivotal Tracker
[#164829201](https://www.pivotaltracker.com/n/projects/2321088/stories/164829201)

### CODE REVIEW

**Review by ValerieRono — APPROVED** [review]
looks good to me Steve

File: authors/apps/authentication/views.py [hunk 1]
```diff
@@ -24,9 +32,49 @@ def post(self, request):
...
+        try:
+            uid = force_text(urlsafe_base64_decode(uidb64))
+            user = User.objects.get(pk=uid)
+        except User.DoesNotExist:
+            user = None
+        
+        if user is not None and email_activation_token.check_token(user, token):
+            user.email_confirmed = True
+            user.save()
+            return Response("Email successfully verified")
```
Comment (ogiste) [inline authors/apps/authentication/views.py]: I would _suggest_ adding key for the message - "Email successfully verified" and any other response messages you send.

**Review by ogiste — DISMISSED** [review]

File: authors/apps/authentication/views.py (hunk 1 above)
Comment (Sm0k3s) [reply authors/apps/authentication/views.py]: Let me work on it.

**Review by koitoror — CHANGES_REQUESTED** [review]
squash commits to 1

**Review by ogiste — APPROVED** [review]
Tests are passing and request body format has also been updated. Looks good.

**Pull request merged**